code/model/SiteTree.php

```diff
@@ -129,21 +128,10 @@ class SiteTree extends DataObject implements PermissionProvider,i18nEntityProvid
 	);
 
 	private static $many_many = array(
-		"LinkTracking" => "SiteTree",
-		"ImageTracking" => "File",
 		"ViewerGroups" => "Group",
 		"EditorGroups" => "Group",
 	);
 
-	private static $belongs_many_many = array(
```

Comment:
All of these relationships are in SiteTreeLinkTracking, so I've simply de-duped them.